packages/tpetra/core/src/Tpetra_CrsGraph_decl.hpp

```diff
@@ -2151,6 +2182,10 @@ namespace Tpetra {
     ///
     void computeLocalConstants();
 
+    /// \brief Forces computation of local triangular properties if they have
+    /// not been computed yet.
+    void computeTriangularProperties();
```

Comment:
Also, please mark this with lots of warning documentation saying "THIS IS FOR EXPERTS ONLY" and "NO PROMISE OF BACKWARDS COMPATIBILITY; THIS METHOD MAY CHANGE OR DISAPPEAR AT ANY TIME."